session_linux.go

```diff
@@ -25,7 +25,7 @@ type paSession struct {
 	sinkInputChannels byte
 }
 
-type masterSession struct {
+type paDevice struct {
```

Comment:
Struct name is a bit inconsistent with other session structs. I suggest using `paDeviceSession` and correspondingly renaming `paSession` -> `paProcessSession`